genesis/spec/presets.go

```diff
@@ -0,0 +1,74 @@
+package spec
+
+import (
+	"fmt"
+
+	"sort"
+
+	"github.com/hyperledger/burrow/permission"
+)
+
+// Files here can be used as starting points for building various 'chain types' but are otherwise
+// a fairly unprincipled collection of GenesisSpecs that we find useful in testing and development
+
```

Comment:
We should add root and developer account types here.

Reply:
my thinking was that the presets functionality was implemented in structure but still something of a stub, but yeah for parity/completeness I'll add those. Note that we can have any manner of compositional presets with multiple accounts or whatever.